wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/Revision.java

```diff
@@ -20,34 +21,148 @@
 @DTO
 public interface Revision {
 
-    /** @return branch name */
+    /**
+     * Parameter which shows that this revision is a fake revision (i.e. TO for Exception)
+     *
+     * @return boolean
```

Comment:
It is not a parameter that i added , so i don't know what is the purpose of this parameter ?
It was there before i added my code